src/plots/layout_attributes.js

```diff
@@ -292,6 +292,16 @@ module.exports = {
             'Sets the background color of the plotting area in-between x and y axes.'
         ].join(' ')
     },
+    axesconvertnumeric: {
```

Comment:
Not sure about the attribute names - in any event I think they should be the same (no need to prefix this one with `axes`) but I'd like to work "strings" into the name if we can do so without making it too long. Maybe just `numericstrings`?

Reply:
What about mentioning `autotype` there as this only impacts that particular function?